src/styles/blocks/header.scss

```diff
@@ -0,0 +1,63 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  height: 60px;
+  &__icon {
+    width: 40px;
+    height: 40px;
+    img {
+      display: block;
+    }
+  }
+
+  &__list {
+    display: flex;
+    align-items: center;
+    font-weight: 500;
+    padding: 0;
+    margin: 0;
+  }
+  &__item {
+    list-style: none;
+    margin-right: 20px;
+    text-decoration: none;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
```

Comment:
The `display: flex;`, `justify-content: center;`, and `align-items: center;` properties are not necessary for the `&__item` class. These properties are used for aligning child elements, but this class does not have any child elements that need to be aligned. Remove these properties.